docs/Using-Fleet/configuration-files/README.md

```diff
@@ -264,15 +270,928 @@ spec:
+      - 342
+      - 98
+  ```
+
+#### server_settings.deferred_save_host
+
+Whether saving host-related information is done synchronously in the HTTP handler of the host's request, or asynchronously. This can provide better performance in deployments with many hosts. Note that this is an **experimental feature** (TODO(mna): is it?).
+
+- Optional setting (boolean).
+- Default value: `false`.
+- Config file format:
+  ```
+  server_settings:
+    deferred_save_host: true
+  ```
+
+### SMTP Settings
+
+<!-- This section replaced a section named SMTP authentication, this ensures links with the #smtp-authentication hash still work -->
+<span id="smtp-authentication" name="smtp-authentication"></span>
+
+#### smtp_settings.authentication_method
+
+The authentication method to use when the authentication type is `authtype_username_password`.
+
+- Optional setting (string).
+- Default value: `authmethod_plain`.
+- Possible values:
+  - `authmethod_cram_md5`
+  - `authmethod_login`
+  - `authmethod_plain`
+- Config file format:
+  ```
+  smtp_settings:
+    authentication_method: authmethod_cram_md5
+  ```
+
+#### smtp_settings.authentication_type
+
+The type of authentication for the configured SMTP server.
+
+- Optional setting (string).
+- Default value: `authtype_username_password`.
+- Possible values:
+  - `authtype_none` - use this if your SMTP server is open
+  - `authtype_username_password` - use this if your SMTP server requires authentication with a username and password
+- Config file format:
+  ```
+  smtp_settings:
+    authentication_type: authtype_none
+  ```
+
+#### smtp_settings.domain
+
+The domain for SMTP. (TODO(mna): what is this? not the host, apparently, as there's also smtp server? Doesn't appear to be used?)
+
+- Optional setting (string).
+- Default value: none.
+- Config file format:
+  ```
+  smtp_settings:
+    domain: example.org
+  ```
+
+#### smtp_settings.enable_smtp
+
+Whether SMTP support is enabled or not to send emails from Fleet.
+
+- Optional setting (boolean).
+- Default value: `false`.
+- Config file format:
+  ```
+  smtp_settings:
+    enable_smtp: true
+  ```
+
+#### smtp_settings.enable_ssl_tls
+
+TODO(mna): doesn't seem to be used?
+Whether to enable SSL/TLS for the SMTP connection.
+
+- Optional setting (boolean).
+- Default value: `true`.
+- Config file format:
+  ```
+  smtp_settings:
+    enable_ssl_tls: false
+  ```
+
+#### smtp_settings.enable_start_tls
+
+Whether to detect if TLS is used by the SMTP server and start using it if so.
+
+- Optional setting (boolean).
+- Default value: `true`.
+- Config file format:
+  ```
+  smtp_settings:
+    enable_start_tls: false
+  ```
+
+#### smtp_settings.password
+
+> **Warning:** Be careful not to store your SMTP credentials in source control. It is recommended to set the password through the web UI or `fleetctl` and then remove the line from the checked in version. Fleet will leave the password as-is if the field is missing from the applied configuration.
```

Comment:
Similar comment to the Jira and Zendesk settings: 

If the best practice is to configure integrations in the UI, I think we can remove the entire Jira and Zendesk section from the docs. This way the docs represent the best practice for using/configuring Fleet.

Reply:
Should I remove SMTP settings and Jira/Zendesk integrations from the default settings too (the example YAML document)?

Reply:
> I think we want the example YAML to reflect the YAML returned when a new user runs fleetctl get config.

Ok then let's leave it as it is, it matches the output of `fleetctl get config`.